### PR TITLE
PXP-474: [CLI] Develop the ability to read annotations in dev config

### PR DESCRIPTION
**Pull request by jk-gan**
## Summary

Ticket: [PXP-474](https://mindvalley.atlassian.net/browse/PXP-474)

## What's Changed

- [x] add the function to read elixir source code and return list of annotations
Screenshot 2023-03-08 at 4 56 59 PM

[PXP-474]: https://mindvalley.atlassian.net/browse/PXP-474?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/69#" title="Author self-approved">jk-gan</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: src/utils/annotation.rs
Comment (onimsha) [inline src/utils/annotation.rs:41]: we should also match the case of `#wukong.mindvalley.dev/config-secrets-location:`

File: src/utils/annotation.rs
Comment (onimsha) [inline src/utils/annotation.rs:1]: given the filename, it should be in plural form, `annotations.rs`. There's no way to tell if we are going to have more annotations in the future.

Comment (jk-gan) [reply src/utils/annotation.rs]: I've renamed it to `annotations.rs`

File: src/utils/annotation.rs
Comment (jk-gan) [reply src/utils/annotation.rs]: alright, I update the pattern so now it will match `#wukong.mindvalley.dev/config-secrets-location:`

**Review by onimsha — APPROVED** [review]

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/69#" title="Author self-approved">jk-gan</a>*, *<a href="https://github.com/mindvalley/wukong-cli/pull/69#pullrequestreview-1336185571" title="Approved">onimsha</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
